src/rulesets/rest/2022-05-25/spectral.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Spectral,
+  RulesetDefinition as SpectralRulesetDefinition,
+} from "@stoplight/spectral-core";
+import { oas } from "@stoplight/spectral-rulesets";
+import { SpectralRule } from "@useoptic/rulesets-base";
+
+const spectral = new Spectral();
```

Comment:
This is just the spectral ruleset pulled out to be a first class citizen (rather than a configurable object)